src/docs/get-started/install/_android-setup.md

```diff
@@ -13,9 +13,21 @@
     This installs the latest Android SDK, Android SDK Command-line Tools,
     and Android SDK Build-Tools, which are required by Flutter
     when developing for Android.
+
 {{site.alert.note}}
-We're updating the flutter tool to use either the Platform-Tools (now obsolete), or the new Command-Line tool if it is available. 
+**IMPORTANT** In Android Studio 3.6 or later, you need to manually add the old
+version of the Android SDK Tools for Flutter to work. To do this:
```

Comment:
For SEO, maybe include the words as shown in the error message?

Also, maybe:
```
 In Android Studio 3.6 (or later) running Flutter 1.12 (stable), 1.14 (beta?), 1.15 (master?), 
  you might see the following error when...
```

As I've shown, please indent the content of the alert by 2 spaces on each line.

Reply:
On the error for SEO, it's a good idea, but it might be confusing at this stage of the instructions. The error occurs when running `flutter doctor` in the next stage, and so a new user won't know they hit the error yet.